src/main/kotlin/telegram/Menu.kt

```diff
@@ -195,65 +169,211 @@ internal fun showRevealMenu(game: Game, bot: Bot, chatId: Long, messageId: Long)
     )
 }
 
-internal fun showAdminListMenu(chatId: Long, messageId: Long, bot: Bot) {
-    bot.editMessageReplyMarkup(
-        ChatId.fromId(chatId),
+internal fun showAdminListMenu(
+    bot: Bot,
+    chatId: Long,
+    messageId: Long,
+    itemsOffset: Int,
+    showNumpadMenu: Boolean = false
+) {
+    val adminsList = admins.find()
+    showPaginatedMenu(
+        chatId,
         messageId,
-        replyMarkup = inlineKeyboard {
-            button(blankCommand named "Список администраторов")
-            admins.find().forEach {
-                accounts.get(it.chatId)?.let { acc ->
-                    row {
-                        button(blankCommand named acc.fullName())
-                        button(removeAdminCommand, it.chatId, messageId)
-                    }
+        bot,
+        "Список администраторов",
+        subListFromOffset(adminsList, itemsOffset, defaultPageSize),
+        adminsList.size,
+        { index, account ->
+            accounts.get(account.chatId)?.let { acc ->
+                row {
+                    button(blankCommand named "${index + 1}. ${acc.fullName()}")
```

Comment:
Number is unhelpful and uses 3+ symbols in a button that's already only half of the screen, which basically hides part of admins name for no reason